Gemfile.lock

```diff
@@ -0,0 +1,210 @@
+GEM
```

Comment:
Example: https://github.com/activerecord-hackery/ransack/pull/755... Some third party gem issue is getting in the middle and now you are forced to either merge a red PR (like I've seen you doing here in the past), hold back the change or start trying to figure out the problem with bundler/third party dependency... no good.